bundles/net.tourbook/src/net/tourbook/map2/view/Map2View.java

```diff
@@ -2974,6 +2977,65 @@ private void onSelectionChanged_TourMarker(final SelectionTourMarker markerSelec
       }
    }
 
+   private void onSelectionChanged_TourPause(final SelectionTourPause pauseSelection, final boolean isDrawSlider) {
+
+//      final TourData tourData = pauseSelection.getTourData();
+//
+//      updateUI_ShowTour(tourData);
+//
+//      final List<Long> allTourMarker = pauseSelection.getSelectedTourPauses();
+//      final int numberOfTourMarkers = allTourMarker.size();
+//
+//      final int leftSliderValueIndex = 0;
+//      final int rightSliderValueIndex = 0;
+
+//      if (tourData.isMultipleTours()) {
+//
+//         if (numberOfTourMarkers == 1) {
+//
+//            leftSliderValueIndex = allTourMarker.get(0).getMultiTourSerieIndex();
+//            rightSliderValueIndex = leftSliderValueIndex;
+//
+//         } else if (numberOfTourMarkers > 1) {
+//
+//            leftSliderValueIndex = allTourMarker.get(0).getMultiTourSerieIndex();
+//            rightSliderValueIndex = allTourMarker.get(numberOfTourMarkers - 1).getMultiTourSerieIndex();
+//         }
+//
+//      } else {
+//
+//         if (numberOfTourMarkers == 1) {
+//
+//            leftSliderValueIndex = allTourMarker.get(0).getSerieIndex();
+//            rightSliderValueIndex = leftSliderValueIndex;
+//
+//         } else if (numberOfTourMarkers > 1) {
+//
+//            leftSliderValueIndex = allTourMarker.get(0).getSerieIndex();
+//            rightSliderValueIndex = allTourMarker.get(numberOfTourMarkers - 1).getSerieIndex();
+//         }
+//      }
+//
+//      if (_isMapSyncWith_Tour || _isMapSyncWith_Slider_One) {
+//
+//         if (isDrawSlider) {
+//
+//            positionMapTo_0_TourSliders(
+//                  tourData,
+//                  leftSliderValueIndex,
+//                  rightSliderValueIndex,
+//                  leftSliderValueIndex,
+//                  null);
+//
+//         } else {
+//
+//            positionMapTo_ValueIndex(tourData, leftSliderValueIndex);
+//         }
+//
+//         keepMapPosition(tourData);
+//      }
+   }
+
```

Comment:
Correct, this will be for my next PR as I wanted this one to focus on the pause slideout and tooltip only.
But if you want me to do it now, let me know

Reply:
For me there is no need to implement it now